src/main/java/competition/subsystems/drive/commands/MoveRightInchByInchCommand.java

```diff
@@ -0,0 +1,17 @@
+package competition.subsystems.drive.commands;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import xbot.common.math.XYPair;
+
+public class MoveRightInchByInchCommand extends SequentialCommandGroup {
+	@Inject
+    MoveRightInchByInchCommand(Provider<SwerveToPointCommand> swerveToPointCommandProvider){
+        var moveLeft = swerveToPointCommandProvider.get();
```

Comment:
Technically you could just put all of this in the OperatorCommandMap, you don't need to wrap it in a SequentialCommandGroup